config/.credo.exs

```diff
@@ -0,0 +1,164 @@
+# This file contains the configuration for Credo and you are probably reading
```

Comment:
Generated with `mix credo gen.config`. I then turned off `Credo.Check.Refactor.MapInto` and `Credo.Check.Warning.LazyLogging` based on warnings the library emitted.

```

info: some checks were skipped because they're not compatible with
your version of Elixir (1.10.3).

You can deactivate these checks by adding this to the `checks` list in your config:

    {Credo.Check.Refactor.MapInto, false},
    {Credo.Check.Warning.LazyLogging, false},
```